src/listeners/messageCreate.ts

```diff
@@ -66,14 +66,42 @@ client.on("messageCreate", async message => {
 		return void (await message.crosspost())
 
 	// Delete non-stringURL messages in review-strings
-	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
-	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings")) {
-		if (!/(https:\/\/)?crowdin\.com\/translate\/hypixel\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi.test(message.content))
-			await message.delete()
-		else {
+	const stringURLRegex = /(https:\/\/)?crowdin\.com\/translate\/\w+\/(?:\d+|all)\/en(?:-\w+)?(?:\?[\w\d%&=$+!*'()-]*)?#\d+/gi
+
+	if (message.channel instanceof TextChannel && message.channel.name.endsWith("-review-strings") && !message.author.bot) {
+		if (!stringURLRegex.test(message.content)) await message.delete().catch(() => null)
+		else if (message.content.match(stringURLRegex)!.length === 1) {
 			await message.react("vote_yes:839262196797669427")
 			await message.react("vote_maybe:839262179416211477")
 			await message.react("vote_no:839262184882044931")
+		} else {
+			const urlsWithComments: string[] = [],
+				contentSplit = message.content.split(" "),
```

Comment:
Need to account for other kinds of spaces
```suggestion
				contentSplit = message.content.split(/\s/).filter(Boolean),
```